include/FxMixerView.h

```diff
@@ -100,6 +100,9 @@ class EXPORT FxMixerView : public QWidget, public ModelView,
 	void moveChannelLeft(int index, int focusIndex);
 	void moveChannelRight(int index);
 
+	// rename the channel
```

Comment:
This comment is useless and should be removed.

Reply:
Yeah, I see what you're saying. I was following the rest of the code there, it looks more out of place without the comment IMO. I could remove all these comments as they are all pretty redundant, but for now I'll just remove the one I added.

![image](https://user-images.githubusercontent.com/6700184/39873662-90c75f58-5439-11e8-8f19-c78f0085da2b.png)